auth.go

```diff
@@ -51,6 +56,15 @@ func (c *Client) renewToken() {
 
 		if err := c.setTokenInfo(); err != nil {
 			c.setStatus("Error renewing token " + err.Error())
+
+			// Try logging in again. If that fails, crash.
+			err := c.setTokenFromAppRole()
+			if err != nil {
```

Comment:
Could convert this to..
```
if err := c.setTokenFromAppRole(); err != nil {
```
.. if you want to keep similar convention in this file